.github/workflows/opened_issue_labeler.yml

```
@@ -16,4 +16,5 @@ jobs:
     - uses: github/issue-labeler@v3.4
       with:
         configuration-path: .github/opened_issue_labeler.yml
+        github-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
```

the currect usage is documented here: https://github.com/github/issue-labeler?tab=readme-ov-file#syncing-labels

```suggestion
        repo-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
```